README.md

```diff
@@ -80,10 +80,14 @@ You can [download](https://github.com/IIvexII/LGU-TimetableAPI/releases) the lat
 
 This software uses the following open source packages:
 
-- [Node.js](https://nodejs.org/)
-- [Express.js](https://expressjs.com/)
-- [Shield.io](https://shields.io/)
-- [JSDOM](https://github.com/jsdom/jsdom)
+- [<img src='https://www.vectorlogo.zone/logos/nodejs/nodejs-ar21.svg' width=64 align='center' >](https://nodejs.org/)
+- [<img src='https://www.vectorlogo.zone/logos/expressjs/expressjs-ar21.svg' width=64 align='center' >](https://expressjs.com/)
+- [<img src='https://www.vectorlogo.zone/logos/shieldsio/shieldsio-official.svg' width=64 align='center' >](https://shields.io/)
+- [<svg xmlns="http://www.w3.org/2000/svg" viewBox="-149 420 100 100" width=32 height=48 >
```

Comment:
Please use the image instead of svg. And also remove the - to make them align horizontally.